Disher/Models/List.h

```diff
@@ -0,0 +1,19 @@
+//
+//  List.h
+//  Disher
+//
+//  Created by Bienn Viquiera on 7/8/22.
+//
+
+#import <Parse/Parse.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface List : PFObject<PFSubclassing>
+
+@property (nonatomic, strong) NSNumber *listID;
+@property (nonatomic, strong) NSArray *recipes;
```

Comment:
suggestion: type-ify your collections. instead of a generic NSArray, do NSArray<Recipe *> (or whatever you're holding on it).